ðŸ‘‹ Hi, Iâ€™m Claudio Lorenzi ( @CloXD ): Bioinformatician, full stack software developer interested in Machine Learning and Artificial Intelligence.

Visit my [webpage](https://cloxd.github.io/CloXD) to know more about me! 

You can find some of my works here:
- [PCaProfiler](https://pcaprofilertest.tk): the Prostate Cancer atlas Profiler, the web tool that enables you to mine clinical and transcriptional data of over 1365 clinical samples across various stages. Associations with prostate cancer progression can be quantified using a quantitative pseudotime progression score. In addition, the tool also allows the annotation of new experimental and clinical data for disease progression.   
- [iMOKA](https://github.com/RitchieLabIGH/iMOKA): a tool to identify k-mers as classifing features from large datasets of sequencing data. The software is mainly written in C++ ( MLPack as ML library ) and Pyhton ( scikit-learn and TF libraries ) for the core part and Angular + Electron for the GUI.
- [IRFinder-S](https://github.com/RitchieLabIGH/IRFinder): a new version of the popular software for the identification and quantification of intron retention events. Includes a CNN based filter that discerns artefactual IR events using visual-like features ( TensorFlow Python library) and a IR database/visualization web tool, [IRbase](https://irbase.igh.cnrs.fr/)
- [MAGeCK_View](https://github.com/CloXD/MAGeCK_view): a small web page that allows to visualize the results of CRISPR screening generated by MAGeCK.
- [Cell2Patients](https://github.com/LucoLab/Villemin_2020): I contributed to my colleague's PhD project that aim to transfer the knowledge aquired from cell lines model to patient data
- [PickPocket](https://github.com/benjaminviart/PickPocket): an attempt to create an automatic protein pocket classifier that uses secondary structure informations alongside with other useful features. Unfortunately, our tests didn't show a real improvement respect to other existing methods, but it was a good attempt.

